scripts/index.js

```diff
@@ -90,23 +123,41 @@ function build(options = {}) {
     //     `${options.fast ? 'fast' : ''}`,
     //     `exclude=${options.exclude.join(',')}`
     // ]
-    const args = ['npm run', 'build-rollup'];
-    let done = false;
-    const task = cp.spawn(args.join(' '), { stdio: 'inherit', shell: true, cwd: wd, env: { ...process.env, MINIFY: Number(!options.fast) } })
-        .on('exit', () => (done = true));
-    return () => {
-        if (!done) {
-            done = true;
-            task.kill();            
-        }
-    };
+    const args = ['npm run', 'build-rollup', '--', options.watch ? '--watch' : ''];
+    let minDest;
+    if (options.destination && !options.fast) {
+        const { name, base, ...rest } = path.parse(path.resolve(options.destination));
+        minDest = path.format({ name: `${name}.min`, ...rest });
+    }
+    return cp.spawn(args.join(' '), {
+        stdio: 'inherit',
+        shell: true,
+        cwd: wd,
+        env: {
+            ...process.env,
+            MINIFY: Number(!options.fast),
+            BUILD_DESTINATION: options.destination,
+            MIN_BUILD_DESTINATION: minDest
+        },
+    });
 }
 
 function startWebsite() {
     if (require(path.resolve(websiteDir, 'package.json')).name !== 'fabricjs.com') {
         console.log(chalk.red('Could not locate fabricjs.com directory'));
     }
-    cp.spawn('npm', ['run', 'start:dev'], {
+    const args = ['run', 'start:dev'];
+
+    //  WSL ubuntu
+    // https://github.com/microsoft/WSL/issues/216 
+    // os.platform() === 'win32' && args.push('--', '--force_polling', '--livereload');
+    if (os.platform() === 'win32') {
```

Comment:
poor suckers
me part of them

Reply:
Did you try wls2?  Windows linux subsystem...

Reply:
Ok is written one line below

Reply:
crossing between kernels is the slowest thing ever 
or something there that slowed down my amazing computer to a halt
I think I should remove it though besides the perf damage it has caused it is really good